### ISSUE TITLE
"Failed getting Magic T value" as off 27 Nov

### ISSUE THREAD

**Issue by Reilsss** [EionRobb/skype4pidgin#501]
Hello, 
as of November 27, I can no longer log into Skype via SkypeWeb Plugin. I run:
- Pidgin Version 2.11.0 on Windows 10
- Skype Web Plugin Version 1.2.2 (from 16 Oct.)

The error I get is "Failed getting Magic T value"

To solve my issue I tryed the follow things without success:
- Connect to Skype from Windows Download Client of Skype = Successful
- Connect to Skype via Windows 10 Skype App = Successful
- Log into Skype Web Site Version = Successful
- Redownload libskypeweb.dll (1.2.2 from 16 Oct.) and connect = No success
- Replace libskypeweb.dll with libskypeweb-debug.dll 1.2.2 and connect = No success
(Btw where does the debug version writes its log file? I cant find it in the Program folder or the appdata folder of pidgin)
- Install Pidgin and libskypeweb.dll 1.2.2 Plugin on a completly diffrent PC = No success
- Try to add `<setting name='refresh-token' type='string'>long string here</setting>` to my account settings XML = No success
(I guess I need to replace "long string here" with something but I do not know what.) 

My settings log like this:
`<account>`
		`<protocol>prpl-skypeweb</protocol>`
		`<name>my Windows Live ID here</name>`
		`<statuses>`
			`<status type='Online' name='Online' active='true'>`
				`<attributes/>`
			`</status>`
			`<status type='Idle' name='Away' active='false'>`
				`<attributes/>`
			`</status>`
			`<status type='Away' name='Not Available' active='false'>`
				`<attributes/>`
			`</status>`
			`<status type='Busy' name='Do Not Disturb' active='false'>`
				`<attributes/>`
			`</status>`
			`<status type='Hidden' name='Invisible' active='false'>`
				`<attributes/>`
			`</status>`
			`<status type='Offline' name='Offline' active='false'>`
				`<attributes/>`
			`</status>`
		`</statuses>`
		`<settings ui='gtk-gaim'>`
			`<setting name='auto-login' type='bool'>0</setting>`
		`</settings>`
		`<current_error>`
			`<type>2</type>`
			`<description>Failed getting Magic T value</description>`
		`</current_error>`
	`</account>`

- I used up till 27th Nov the Microsoft Live login variant with the @ in the account. (I use this on the Skype desktop client and the website as well.
- I also tryed my Skype ID (the one that has not the @ in it) but that didnt change anything.

I also removed the "AppData\Roaming\.purple\certificates\x509\tls_peers" the login.live.com and login.skype.com files to force Pidgin to get those files new.

Still I get the "Failed getting Magic T value" error.

Is there anything else I can try out? Does the plugin / pidgin cache somewhere else data that I should maybe clean? So far I only know about the .purple folder in AppData and the Program installation Directory.

I havent tryed changing my Skype password on the skpye client/site yet. But I'm not sure if that would
change anything on getting the token considering that it works fine on web and the orginal clients.

Anything else I could try out?

**Comment by EionRobb** [comment]
Just wanted to double check, did you follow the flow chart at https://github.com/EionRobb/skype4pidgin/tree/master/skypeweb#issues ?

If so, can you get a debug log (from the buddy list,  Help->Debug Window) as you reconnect to the account and paste it here (always a good idea to check for personal details/username/passwords to clear out of the log before attaching)

**Comment by Reilsss** [comment]
Hello EionRobb,
thank you so much for responding. In short, I can connect to Skype via your plugin now.
Here what I did between last night asking and this eveing.
1. I logged into web.skype.com from my work PC at the office (connects to internet by diffrent router), with success.
2. I downloaded Pidgin 1.2.2 and your skype web installer as well as the debug dll.
3. Installed pidgin on the work PC and the skype plugin via installer. And configurated my account with the windows live  ID (the one with @) And tryed to connect with my password. This worked successful.
4. I replaced the normal skype plugin dll with the debug one, to get a log file as you descripted above. (worked also fine).
(5. Saved the log and my config files to a USB stick to compare with what I get at home.)
6. At home I started pidgin and tryed to reenable the skype plugin, (with did fail the past 2 days with mentioned error). And for some magic reasons it just worked. Like it fixed it self.

Well I guess logging into pidgin skype web plugin from my office work computer did somehow reset the toke maybe (that or logging into web.skype.com from that computer). Otherwise I do not know why It did not work the past days and now it suddenly does.

I want to appology for oping this issue, now that it kind of got fixed by itself. It makes me feel like I waisted your time. Your responce and the situation anyhow made me realse how much I like your project. I will go give you a tipjard, its the lease I can do to thank you for your hard work and effords.

I guess the issue can be closed or deleted, as it kinda not a issue now. So sorry for this again, as well as thank you for your advice.
